github.js

```diff
@@ -31,19 +32,31 @@ async function constructPrMsg(sg, branch) {
  * @param {Object.<string, {title: string, pr: number}>} branchToPrDict
  * @param {string} currentBranch
  * @param {string} combinedBranch
+ * @param {'text'|'table'} format
  */
-function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch) {
-  let contents = '<pr-train-toc>\n\n#### PR chain:\n';
-  contents = Object.keys(branchToPrDict).reduce((output, branch) => {
-    const maybeHandRight = branch === currentBranch ? '👉 ' : '';
+function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch, format) {
+  let contents = '<pr-train-toc>\n\n';
+  let tableData = [['', 'PR', 'Description']];
+  Object.keys(branchToPrDict).forEach((branch) => {
+    const maybeHandRight = branch === currentBranch ? '👉 ' : ' ';
     const maybeHandLeft = branch === currentBranch ? ' 👈 **YOU ARE HERE**' : '';
     const combinedInfo = branch === combinedBranch ? ' **[combined branch]** ' : ' ';
-    output += `${maybeHandRight}#${branchToPrDict[branch].pr}${combinedInfo}(${branchToPrDict[
-      branch
-    ].title.trim()})${maybeHandLeft}`;
-    return output + '\n';
-  }, contents);
-  contents += '\n</pr-train-toc>';
+    const prTitle = branchToPrDict[branch].title.trim();
+    const prNumber = `#${branchToPrDict[branch].pr}`;
+    const prInfo = format === 'text'
+      ? `${combinedInfo}(${prTitle})`
+      : `${combinedInfo}${prTitle}`.trim();
+    const parts = [maybeHandRight, prNumber, prInfo, format === 'text' && maybeHandLeft].filter(Boolean);
+    if (format === 'text') {
+      contents += parts.join('') + '\n';
+    } else {
+      tableData.push(parts);
+    }
+  });
+  if (format === 'table') {
+    contents += table(tableData, { stringLength: width }) + '\n';
```

Comment:
Using the [`string-width`](https://github.com/sindresorhus/string-width) module here works around the strange rules for the width of emoji characters to end up with a table with much more aligned delimiters between columns.